### PR TITLE
You need to append /external to the clone path, otherwise it is throw…

### PR DESCRIPTION
**Pull request by jkblume**
…ing an error

### CODE REVIEW

**Comment by marek-mazur** [comment]
it is true

File: README.md [hunk 1]
```diff
@@ -7,12 +7,12 @@ and send a pull request on GitHub.
 
 To install the extensions in user space type the following in your terminal:
 ```bash
-git clone git@github.com:albertlauncher/external.git "$HOME/.local/share/albert/"
+git clone git@github.com:albertlauncher/external.git "$HOME/.local/share/albert//external/"
```
Comment (ManuelSchneid3r) [inline README.md]: dup slashes

**Comment by marek-mazur** [comment]
Why it is still not merged?

File: README.md (hunk 1 above)
Comment (ManuelSchneid3r) [inline README.md]: Damn I should learn to use the github review tool. This should be a change request.

**Comment by ManuelSchneid3r** [comment]
Just quickfixed it myself. Thank you.

**Pull request closed without merging**